README.md

```diff
@@ -4,6 +4,24 @@ It's possible to run BakkesMod on Linux with an up to date version of Rocket Lea
 
 Using Proton is only recommended if you're running Rocket League through Steam. If that's not the case using Wine or [Wine-GE](https://github.com/GloriousEggroll/wine-ge-custom) is advised as using Proton outside of Steam can cause issues.
 
+## Steam Deck
+
+You will need to download `protontricks` from flatpak, which can be done through the discover app. Installing `GE-Proton` is also a good idea as well. 
```

Comment:
`protontricks` probably refers to https://flathub.org/apps/com.github.Matoking.protontricks?